command_line/find_bad_pixels.py

```diff
@@ -30,6 +32,9 @@
 image_range = None
   .type = ints(value_min=0, size=2)
   .help = "Image range for analysis e.g. 1,1800"
+nproc = 1
+  .type = int
```

Comment:
```suggestion
  .type = int(value_min=1)
```
We can't have fewer than 1 process